railties/lib/rails/generators/rails/app/templates/gitignore.tt

```diff
@@ -35,3 +35,4 @@
 
 /public/assets
 <% end -%>
+/config/credentials/*.key
```

Comment:
Rather than adding it here, why don't we generate it like we do with the master key?

https://github.com/rails/rails/blob/951ea8f75d9bf6d2999df2f0497347976dce9476/railties/lib/rails/generators/rails/app/app_generator.rb#L191-L205

https://github.com/rails/rails/blob/951ea8f75d9bf6d2999df2f0497347976dce9476/railties/lib/rails/generators/rails/master_key/master_key_generator.rb#L35-L41

Reply:
I changed it to use that. It took slightly more code, but I made it in a separate commit so I can revert if the previous version is preferred.